standard/namespaces.md

```diff
@@ -669,7 +713,9 @@ Using this notation, the meaning of a *qualified_alias_member* is determined as
 > *Note*: The identifier `global` has special meaning only when used as the left-hand identifier of a *qualified_alias_name*. It is not a keyword and it is not itself an alias; it is a contextual keyword ([§7.4.4](lexical-structure.md#744-keywords)). In the code:
 > ```csharp
 > class A { }
-> class C {
+>
+> class C
```

Comment:
This was one of 2 instances for K&R braces on `class` declarations. That's why I changed this one.